sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_download_stream.py

```diff
@@ -68,7 +68,7 @@ async def __anext__(self) -> bytes:
         try:
             return await self._yield_data()
         except StopAsyncIteration:
-            if self._downloaded >= self._blob_size:
+            if self._downloaded == self._blob_size:
```

Comment:
do we already have test cases that cover this?

Reply:
Reverted this update. Yes, we have. There's digest validation tests, it will check by digest when it stops downloading.